components/admin_console/team_channel_settings/channel/details/channel_modes.jsx

```diff
@@ -15,7 +15,9 @@ const SyncGroupsToggle = ({isSynced, isPublic, onToggle}) => (
     <LineSwitch
         toggled={isSynced}
         last={isSynced}
-        onToggle={() => onToggle(!isSynced, isPublic)}
+        onToggle={() => {
+            onToggle(!isSynced, isPublic, true);
```

Comment:
this change is no longer needed, you keep passing the `true` here but nothing is handling it.